code/modules/mob/dead/new_player/sprite_accessories/ears.dm

```diff
@@ -171,6 +171,12 @@
 	color_src = MATRIXED
 	icon = 'modular_citadel/icons/mob/mam_ears.dmi'
 
+/datum/sprite_accessory/ears/human/rabbit
+	name = "Rabbit Lop-Eared"
+	icon_state = "rabbitlop"
+	color_src = MATRIXED
+	icon = 'modular_citadel/icons/mob/mam_ears.dmi'
```

Comment:
make it modular

Reply:
Honestly not exactly sure what that means, but I tried to do it anyways.